recipe/build.sh

```diff
@@ -8,7 +13,6 @@ fi
             --enable-tempstore \
             --enable-shared=yes \
             --disable-tcl \
-            --disable-readline \
```

Comment:
So there is an `--enable-readline` flag, which we should try. We can also try `--disable-libedit`.